cmd/uhc/account/status/cmd.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/openshift-online/uhc-cli/cmd/uhc/account/users"
```

Comment:
Please move the function into a package (or make a new one) and use it for both the `status` & `users` commands.